README.md

```diff
@@ -0,0 +1,112 @@
+# Microservice Utils
+
+[![Build Status](https://travis-ci.org/Cimpress-MCP/microservice-utils.js.svg?branch=master)](https://travis-ci.org/Cimpress-MCP/microservice-utils.js)
+[![npm version](https://badge.fury.io/js/microservice-utils.svg)](https://www.npmjs.com/package/microservice-utils)
+
+Utilities supporting authorization, request logging and other often required parts of microservices.
+
+## Using this library
+
+Install the library.
+
+```bash
+npm install microservice-utils
+```
+
+### Request Logger
+Logger which takes care of truncating Bearer tokens and safe JSON stringification. While by default it logs to console,
+it supports a custom logging function (e.g. to SumoLogic).
+```javascript
+const { RequestLogger } = require('microservice-utils');
+let requestLogger = new RequestLogger({ loggerFunc: console.log });
+requestLogger.log({ title: 'Message title', level: 'WARN', error: 'Error'});
+``` 
+
+It can be also configured to extend the `Error` object using 
+[error-object-polyfill.js](https://github.com/wparad/error-object-polyfill.js), which enables stringification of `Error`
+objects to JSON, so that the error message and stack trace appears in logs.
+```javascript
+let requestLogger = new RequestLogger({ loggerFunc: console.log, extendErrorObjects: true });
```

Comment:
I'd be tempted to default this to `true` instead of `false`. Who doesn't want that?

Reply:
Defaulted to `true`.